pom.xml

```diff
@@ -700,6 +701,11 @@
                 <artifactId>commons-cli</artifactId>
                 <version>${version.commons-cli}</version>
             </dependency>
+            <dependency>
+                <groupId>commons-codec</groupId>
+                <artifactId>commons-codec</artifactId>
+                <version>${version.commons-codec}</version>
+            </dependency>
```

Comment:
@ehsavoie Have you tested whether it is just possible to exclude `commons-codec` transitive dependency from  `org.eclipse.jgit`? It seems it is not used at runtime (it has not been declared for org.eclipse.jgit module.xml) and we do not compile `org.eclipse.jgit`, so I guess excluding it would be enough.